docs/user-guide/config-examples.md

```diff
@@ -20,6 +20,7 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 | [hbjydev/hvim] | |
 | [MikaelFangel/nixvim-config] | An easy-setup configuration for NixVim, focused on straightforward customization |
 | [nicolas-goudry/nixvim-config] | Heavily inspired by AstroNvim |
+| [niksingh710/nvix] | Ported from the previous lazy based raw config. Contains most of the use cases. (Custom plugins/Nixpkgs plugins/Nixvim Plugins are all covered) |
```

Comment:
Sorry to nitpick, I think something like this would read better?

```suggestion
| [niksingh710/nvix] | Ported from a lazy-based lua config. Contains examples of most use cases, including custom, nixpkgs, & nixvim plugins. |
```

Reply:
I agree that my initial attempt was somewhat verbose, resulting in a lengthy sentence. Your version is much more concise and polished.